tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -1140,16 +1152,16 @@ private void validateTitleChanged(String expectedTitle, Runnable browserSetFunc)
 	browserSetFunc.run();
 	shell.open();
 
-	boolean hasFinished = waitForPassCondition(() -> actualTitle.get().length() != 0
-			&& !actualTitle.get().contains("about:blank")); // Windows sometimes does 2 loads, one "about:blank", and one actual load.
-	boolean passed = hasFinished && actualTitle.get().equals(expectedTitle);
+	boolean passed = waitForPassCondition(() -> actualTitle.get().equals(expectedTitle));
 	String errMsg = "";
-	if (!hasFinished)
-		errMsg = "Test timed out. TitleListener not fired";
-	else if (!actualTitle.get().equals(expectedTitle)) {
-		errMsg = "\nExpected title and actual title do not match."
-				+ "\nExpected: " + expectedTitle
-				+ "\nActual: " + actualTitle;
+	if (!passed) {
+		if (actualTitle.get().length() == 0) {
+			errMsg = "Test timed out. TitleListener not fired";
+		} else {
+			errMsg = "\nExpected title and actual title do not match."
+					+ "\nExpected: " + expectedTitle
+					+ "\nActual: " + actualTitle;
+		}
 	}
 	assertTrue(errMsg + testLog.toString(), passed);
```

Comment:
```suggestion
	assertEquals(testLog.toString(), expectedTitle, actualTitle.get());
```

Reply:
But then we cant say if the expected title was wrong or it was a time out. there was errMsg added in the previous version of this test as well.

Reply:
Done